readme.md

```diff
@@ -102,8 +102,8 @@ GitHub Enterprise is also supported: [How to enable it](https://fregante.github.
 			<p><img src="https://user-images.githubusercontent.com/1402241/109592127-5f922200-7ad4-11eb-8dfa-1d80fb28e70e.png">
 	<tr>
 		<th width="50%">
-			<p><a title="linkify-code"></a> Linkifies issue/PR references and URLs in code and conversation titles
-			<p><img src="https://cloud.githubusercontent.com/assets/170270/25370217/61718820-29b3-11e7-89c5-2959eaf8cac8.png">
+			<p><a title="linkify-code"></a> Lets you subscribe to opening/opening closing events of issues in one click.
+			<p><img src="https://github.com/refined-github/refined-github/assets/1402241/cbc98b51-d173-40c6-b21e-5f0bae3d800c">
```

Comment:
Old features lose the highlight… especially because they're breaking 🥲 

- #6336

Reply:
So this URL redirects to `https://github-production-user-asset-6210df.s3.amazonaws.com/1402241/238186901-cbc98b51-d173-40c6-b21e-5f0bae3d800c.png`

Should we just use the final URL? I think that as long as we have the original image uploaded to the issue, we can use the final .png URL (which is why the feature verification is failing)